source/content/guides/wordpress-composer/04-bedrock-composer.md

```diff
@@ -0,0 +1,75 @@
+---
+title: Using Bedrock to Create Your WordPress Pantheon Site
```

Comment:
In general, I don't think we should be proposing people to use Bedrock upstream yet as it's not production-ready and development is still pending/in-progress.